contracts/GolemNetworkToken2.sol

```diff
@@ -0,0 +1,17 @@
+// Copyright 2018 Golem Factory
+// Licensed under the GNU General Public License v3. See the LICENSE file.
+
+pragma solidity ^0.4.19;
+
+import "./ExtendedToken.sol";
+import "./TokenProxy.sol";
+
+contract GolemNetworkToken2 is TokenProxy, ExtendedToken {
+
+    string public constant name = "Golem Network Token 2";
+    string public constant symbol = "GNT2";
```

Comment:
Same here. GNTB, GWBT, whatever. We have alpha2 and golem 2.0 - that's enough.

Reply:
Maybe Golem Network Token Whatever then?